components/TransactionForm.js

```diff
@@ -116,13 +117,21 @@ export default function TransactionForm({
                   <option value="" disabled>
                     Please select a category
                   </option>
-                  {categories.map((category) => (
-                    <option key={category.id} value={category.name}>
-                      {category.name}
-                    </option>
-                  ))}
+                  {categories
+                    .sort((a, b) => a.name.localeCompare(b.name))
+                    .map((category) => (
+                      <option key={category.id} value={category.name}>
+                        {category.name}
+                      </option>
+                    ))}
                 </StyledCategorySelect>
-                <StyledLink href={"/settings"} aria-label="Settings">
+                <StyledLink
+                  href={"/settings"}
+                  aria-label="Settings"
+                  onClick={() => {
+                    closeModal();
```

Comment:
we found a way that is better/easier than just using a modal: une "dialog" https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

maybe this helps 🤷‍♂️

Reply:
We build a Modal component and i had to insert the closeModal() into the link to make sure, the modal on the settings page, where I am being redirected to via the link, is closed. This solution was the only one that worked.

We will take a closer look at your suggestion though, as it does look interesting for us. Thanks!